packages/reakit/src/Box/__tests__/Box-test.tsx

```diff
@@ -10,3 +11,10 @@ test("render", () => {
     </div>
   `);
 });
+
+test("has no a11y violations", async () => {
+  const { container } = render(<Box>box</Box>);
+  const results = await verify(container.innerHTML);
+
+  expect(results).toHaveNoViolations();
```

Comment:
Nice! 👍 

I think we can do that for the example tests in follow-up PRs. Those have more real use cases.